vochain/process.go

```diff
@@ -335,7 +360,7 @@ func (v *State) GetProcessResults(pid []byte) ([][]string, error) {
 }
 
 // SetProcessCensus sets the census for a given process, only if that process enables dynamic census
-func (v *State) SetProcessCensus(pid, censusRoot []byte, censusURI string, commit bool) error {
+func (v *State) SetProcessCensus(pid, censusRoot []byte, censusURI string, txSender common.Address, commit bool) error {
```

Comment:
The state methods which are not "tx" related (such as SetProcessCensus, SetProcessStatus, AddVote, etc...) should be agnostic to the transaction based checks.

The only checks that these methods shall do are State related, i.e the process exist? But not the cost or other transaction related stuff. These checks go to specific-scope check transaction methods such as `SetProcessCensusTxCheck()`

Reply:
This comment applies to all other methods.